docs/prow/production-cluster-configuration.md

```diff
@@ -60,7 +60,31 @@ Use the following tools and configuration:
    export ENCRYPTION_KEY_NAME=kyma-prow-encryption
    ```
 
-4. Run the following script to start the installation process:
+4. Prepare the workload cluster before installing Prow by running:
+
+    ```bash
+      export WORKLOAD_CLUSTER_NAME=kyma-prow-workload
+      export ZONE=europe-west3-a
+      export PROJECT=sap-kyma-prow
+
+      ### In GKE get KUBECONFIG for cluster kyma-prow-workload
+      gcloud container clusters get-credentials $WORKLOAD_CLUSTER_NAME --zone=$ZONE --project=$PROJECT
+
+      ./set-up-workload-cluster.sh
+    ```
+
+    This script performs the following steps:
+    - Creates a ClusterRoleBinding to provide access to the Prow cluster. This way it enables running and monitoring jobs on the workload cluster.
+    - Creates Kubernetes `Secrets` resources out of Secrets fetched from the GCP bucket.
+
+5. >**NOTE:**: Create the workload cluster beforehand and make sure the **local** kubeconfig for the Prow admin contains the context for this cluster. Point the **current** kubeconfig to the Prow cluster.
+
+Run the following script to create a Secret. This way you allow the Prow cluster to access the workload cluster: 
+    ```bash
+    ./create-secrets-for-workload-cluster.sh
+    ```
```

Comment:
Could you check if this code block renders correctly? Because I can see it all in one line, and not in the codeblock.

Reply:
It works but to be extra sure, I added another line break.